### ISSUE TITLE
API key, not password

### ISSUE THREAD

**Issue by divenal** [twonk/MyEnergi-App-Api#24]
Hi,
   perhaps make it clearer that authentication is the generated API key, not the app account password.

I created the API key on the website - couldn't actually find a way to do so on the app, so not sure whether the app calls it a password or key. But I wasted a bit of time using the wrong password.

**Comment by rob0101** [comment]
Yup, I was using a password for ages, then another integration asked me to setup a Myenergi API key and then my existing scripts broke until I found this comment, swapped my old password to the new api key and things worked again.
